### PR TITLE
Update scsi controller on ppc vms to support scsi hotplug

### PR DESCRIPTION
**Pull request by chloerh**
Since hotplug scsi disk with ibmvscsi controller is not supported,
replacing with virtio-scsi model could fix it.

Signed-off-by: Haijiao Zhao <haizhao@redhat.com>

### CODE REVIEW

**Comment by chloerh** [comment]
Test result

PASS 1-type_specific.io-github-autotest-libvirt.virsh.attach_device.block.single_scsi_file.without_alias.normal_test.hot_attach_hot_vm.persistent.name_ref.file_positional.domain_positional

File: libvirt/tests/src/virsh_cmd/domain/virsh_attach_device.py [hunk 1]
```diff
@@ -888,6 +889,22 @@ def remove_non_disks(vm_name, vmxml):
...
+    :param vm_name: domain name
+    :param vmxml: domain xml
+    :return:
+    """
+    machine = platform.machine().lower()
+    if 'ppc64le' in machine:
+        # Remove old scsi controller, replace with virtio-scsi controller
+        vmxml.del_controller('scsi')
+        vmxml.sync()
+        virsh.start(vm_name, debug=True, ignore_status=False)
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/domain/virsh_attach_device.py:905]: Do you need judge whether previous VM state is in running, rather that start it directly regardless of previous VM state?

Comment (chloerh) [reply libvirt/tests/src/virsh_cmd/domain/virsh_attach_device.py:905]: @chunfuwen Actually `vmxml.sync()` will destroy vm, therefore vm needs to be started after `sync()`

**Review by Yingshun — APPROVED** [review]

File: libvirt/tests/src/virsh_cmd/domain/virsh_attach_device.py (hunk 1 above)
Comment (chunfuwen) [reply libvirt/tests/src/virsh_cmd/domain/virsh_attach_device.py:905]: I mean if previous VM is not running, then at last you try to start it due to this change. In this way, the test logic could be changed.

Comment (chloerh) [reply libvirt/tests/src/virsh_cmd/domain/virsh_attach_device.py:905]: @chunfuwen I guess you haven't read code coming after these lines

```
    if test_params.start_vm:
        # Make sure VM is working
        test_params.main_vm.verify_alive()
        test_params.main_vm.wait_for_login().close()
    else:  # VM not suppose to be started
        if test_params.main_vm.is_alive():
            test_params.main_vm.destroy(gracefully=True)
```

This piece of code makes sure that what you're worrying won't happen.

**Review by chunfuwen — APPROVED** [review]
LGTM

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**